cdk/domino_cdk/config/template.py

```diff
@@ -80,7 +80,7 @@ def add_nodegroups(name, count, min_size, instance_types, labels, disk_size=100,
         "gpu",
         gpu_nodegroups,
         0,
-        ["p3.2xlarge"],
+        ["g4dn.xlarge"],
```

Comment:
unless we're comfortable addressing PLAT-2441, we should do
```
["g4dn.xlarge" if dev_defaults else "p3.2xlarge"]
```

Reply:
or we use larger g4dn instances which are also cheaper than p3.